packages/jbrowse-plugin-apollo/src/ApolloTextSearchAdapter/ApolloTextSearchAdapter.ts

```diff
@@ -0,0 +1,72 @@
+import { readConfObject } from '@jbrowse/core/configuration'
+import {
+  BaseAdapter,
+  BaseTextSearchAdapter,
+  BaseTextSearchArgs,
+} from '@jbrowse/core/data_adapters/BaseAdapter'
+import BaseResult from '@jbrowse/core/TextSearch/BaseResults'
+import { UriLocation } from '@jbrowse/core/util'
+import { getFetcher } from '@jbrowse/core/util/io'
+
+export class ApolloTextSearchAdapter
+  extends BaseAdapter
+  implements BaseTextSearchAdapter
+{
+  get baseURL() {
+    return readConfObject(this.config, 'baseURL').uri
+  }
+
+  get trackId() {
+    return readConfObject(this.config, 'trackId').uri
+  }
+
+  get internetAccountPreAuthorization():
+    | { authInfo: { token: string }; internetAccountType: string }
+    | undefined {
+    return readConfObject(this.config, 'baseURL')
+      .internetAccountPreAuthorization
+  }
+
+  async searchFeatures(term: string) {
+    const url = new URL(`features/searchFeatures?term=${term}`, this.baseURL)
```

Comment:
It's probably good to use `URLSearchParams` to create this so it can e.g. handle conversion of characters (like spaces).
```suggestion
    const url = new URL('features/searchFeatures', this.baseURL)
    const searchParams = new URLSearchParams({ term })
    url.search = searchParams.toString()
term=${term}`, this.baseURL)
```